services/headless-lms/models/src/library/peer_or_self_reviewing.rs

```diff
@@ -283,7 +283,12 @@ pub async fn create_peer_or_self_review_submission_for_user(
 
     tx.commit().await?;
 
-    let _ = check_if_exercise_needs_reset_after_zero_points_from_review(
```

Comment:
wrap where create_peer_or_self_review_submission_for_user is called with a transaction.